unabridged-works-volume-1/original/03-way-of-life.adoc

```diff
@@ -1908,9 +1965,13 @@ popery and episcopacy was innocent before the Lord,
 though accused by them as guilty of error in itself,
 and of disturbance to the public peace, as this is now by thee.
 
-=== SOME PRINCIPLES GUIDING OUT OF THE APOSTASY, TO THE CHRISTIAN SPIRIT AND LIFE AGAIN
+=== Some Principe Guiding out of the Apostasy, to the Christian Spirit and Life Again.
```

Comment:
Looks like you mis-typed `Principles`.  Probably also remove the period that you introduced here too, in general I usually default to not adding periods on long titles (although occasionally i do). 👍

Reply:
haha so weird I did that.